repy.py

```diff
@@ -74,10 +74,6 @@
 
 from exception_hierarchy import *
 
-# BAD: REMOVE these imports after we remove the API calls
-#import emulfile
-import emulmisc
```

Comment:
emulmisc was only needed for:
`usercontext["getlasterror"] = emulmisc.getlasterror`, which was removed in this commit.